src/devices/esp8266.c

```diff
@@ -329,6 +330,9 @@ bool esp8266_init(struct Serial *s, const size_t max_cmd_len,
         if (!_setup(s, max_cmd_len))
                 return false;
 
+        /* Hard reset our ESP8266*/
```

Comment:
In MK2, there's no IO configuration, and the hard device reset is a NOOP, so I think this should be OK?